### PR TITLE
docs: add examples of sending transactions

### PR DESCRIPTION
**Pull request by helciofranco**
Add a `Sending Transaction` section to the Wallet docs.

| 📷 Demo |
| --- |
| Screenshot 2024-06-03 at 21 22 58 |

### CODE REVIEW

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
can we add the `useSendTransaction` in the "Hooks Reference" session as well?

File: packages/docs/package.json [hunk 1]
```diff
@@ -15,7 +16,7 @@
     "@fuel-ui/icons": "0.23.3",
     "@fuel-ui/react": "0.23.3",
     "@fuels/connectors": "0.4.0",
-    "@fuels/react": "0.20.0",
+    "@fuels/react": "0.21.0-main-30f37c8",
```
Comment (LuizAsFight) [inline packages/docs/package.json:19]: can we use `0.21.0` directly? it's being released already

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/docs/package.json (hunk 1 above)
Comment (helciofranco) [reply packages/docs/package.json:19]: yes, for sure. I'll update it here

Comment (helciofranco) [reply packages/docs/package.json]: https://github.com/FuelLabs/fuels-wallet/pull/1346/commits/a20145a09fc917e48a831027188d572170687ce7

**Comment by helciofranco** [comment]
@LuizAsFight changes have been pushed

https://github.com/FuelLabs/fuels-wallet/pull/1346/commits/da7f8960bad665abc5694070772bea888dc00a86

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**